proposals/dao/end_tribe_incentives.ts

```diff
@@ -0,0 +1,145 @@
+  ).to.equal(true);
+
+  expect(
+    await contracts.core.hasRole(ethers.utils.id('TRIBAL_CHIEF_ADMIN_ROLE'), addresses.tribalChiefSyncV2)
+  ).to.equal(false);
+
+  expect(await contracts.core.hasRole(ethers.utils.id('FUSE_ADMIN'), addresses.tribalChiefSyncV2)).to.equal(false);
+  expect(await contracts.core.hasRole(ethers.utils.id('VOTIUM_ADMIN_ROLE'), addresses.opsOptimisticTimelock)).to.equal(
+    false
+  );
+
+  // 1. Verify TribalChief block rewards are effectively 0
+  expect(await tribalChief.tribePerBlock()).to.equal(NEW_TRIBE_BLOCK_REWARD);
+
+  // 2. Validate number of pools is as expected
+  const poolIds = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17];
+  const numPools = await tribalChief.numPools();
+  expect(numPools).to.equal(poolIds.length);
+
+  // 3. Validate that all pool AP points are set to zero, apart from Fei-Tribe Uniswap V2
+  const feiRariPoolId = '0';
+  for (const pid in poolIds) {
+    const poolInfo = await tribalChief.poolInfo(pid);
+
+    if (pid == feiRariPoolId) {
+      expect(poolInfo.allocPoint).to.be.equal(NEW_TOTAL_ALLOC_POINTS.toString());
+    } else {
+      expect(poolInfo.allocPoint).to.be.equal('0');
+    }
+  }
+
+  // 4. Verify rewardsDistributorAdmin internal ACL revoked
+  const rewardsDistributorAdmin = contracts.rewardsDistributorAdmin;
+  expect(
+    await rewardsDistributorAdmin.hasRole(
+      ethers.utils.id('AUTO_REWARDS_DISTRIBUTOR_ROLE'),
+      addresses.feiDaiAutoRewardsDistributor
+    )
+  ).to.equal(false);
+
+  expect(
+    await rewardsDistributorAdmin.hasRole(
+      ethers.utils.id('AUTO_REWARDS_DISTRIBUTOR_ROLE'),
+      addresses.feiUsdcAutoRewardsDistributor
+    )
+  ).to.equal(false);
+
+  expect(
+    await rewardsDistributorAdmin.hasRole(
+      ethers.utils.id('AUTO_REWARDS_DISTRIBUTOR_ROLE'),
+      addresses.autoRewardsDistributor
+    )
+  ).to.equal(false);
+
+  expect(
+    await rewardsDistributorAdmin.hasRole(
+      ethers.utils.id('AUTO_REWARDS_DISTRIBUTOR_ROLE'),
+      addresses.d3AutoRewardsDistributor
+    )
+  ).to.equal(false);
+
+  expect(
+    await rewardsDistributorAdmin.hasRole(
+      ethers.utils.id('AUTO_REWARDS_DISTRIBUTOR_ROLE'),
+      addresses.fei3CrvAutoRewardsDistributor
+    )
+  ).to.equal(false);
+
+  // 5. Verify total AP points is 1
+  expect(await tribalChief.totalAllocPoint()).to.be.equal(NEW_TOTAL_ALLOC_POINTS.toString());
+
+  // 6. Verify an AutoRewardDistributor speed is set correctly
+  const d3Ctoken = await contracts.rariPool8Comptroller.cTokensByUnderlying(addresses.curveD3pool);
+
+  const d3RewardSpeed = await contracts.d3AutoRewardsDistributor.getNewRewardSpeed();
+  expect(d3RewardSpeed[1]).to.equal(false); // updateNeeded
+  expect(d3RewardSpeed[0]).to.equal(0); // newSpeed
+
+  const delegatorReportedSpeed = await contracts.rariRewardsDistributorDelegator.compSupplySpeeds(d3Ctoken);
+  expect(delegatorReportedSpeed).to.equal(0);
+};
```

Comment:
you should verify here that all AutoRewardsDistributors are set to 0 alloc points as well.